app/advancedfilters/advancedFilters.js

```diff
@@ -72,7 +72,7 @@ app.controller('AdvancedFiltersCtrl', function($scope, $rootScope, $routeParams,
       var gps = stringService.getTextareaItemsAsArray($scope.gpTextArea);
       angular.forEach(gps, function(gpID){
         if(filteringService.validateGeneProduct(gpID)) {
-          filteringService.addFilter('gpID',gpID,true);
+          $scope.filters.gpID[gpID] = true;
```

Comment:
Not sure if the problem is here but still when I try annotations?gpID=T1DG24 and click Gene Product Ids, I get no checkbox with T1DG24 as label.